build/dependencies.props

```diff
@@ -131,7 +131,6 @@
     <SystemBuffersPrimitivesPackageVersion>0.1.0-preview2-180130-1</SystemBuffersPrimitivesPackageVersion>
     <SystemCodeDomPackageVersion>4.5.0-preview2-26130-01</SystemCodeDomPackageVersion>
     <SystemCollectionsImmutablePackageVersion>1.5.0-preview2-26126-03</SystemCollectionsImmutablePackageVersion>
-    <SystemCollectionsSequencesPackageVersion>0.1.0-preview2-180130-1</SystemCollectionsSequencesPackageVersion>
```

Comment:
This was removed in release/2.1, should it be removed in dev?

Reply:
Yes